samples/matter/thermostat/prj_release.conf

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# This sample uses Kconfig.defaults to set options common for all
+# samples. This file should contain only options specific for this sample
+# or overrides of default values.
+
+# Enable CHIP
+CONFIG_CHIP=y
+CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y
+CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+# 32782 == 0x8000 (example Product ID added temporaly,
```

Comment:
Moreover, the note with the link below has been created only for the Template sample because in Matter there is no Template PID. In the case of the Thermostat device type, we can change it to the proper value and remove the note.